### ISSUE TITLE
Docker Image exit with missing debugpy

### ISSUE THREAD

**Issue by gandalfb** [AnotherDaniel/smahub#22]
Thank you for this Hub. It really helps to integrate the data.

I tried to deploy the docker-compose and stumbled accross the container not starting.
The change to pipx/venv seems to break the execution.

I was able to get it running by adjusting Dockerfile and the start script.
If this is "best bractice", I am not quite sure.

Dockerfile
```
FROM alpine:3

# Prepare base system
RUN apk update && apk upgrade && \
    apk add python3 py3-pip git py3-virtualenv

# Install smahub
RUN mkdir -p /opt/smahub
WORKDIR /opt/smahub

ENV PATH=/opt/venv/bin:$PATH
RUN mkdir /opt/venv && \
    virtualenv /opt/venv

# RUN git clone --depth=1 --branch=main https://github.com/AnotherDaniel/smahub . &>
COPY . .
RUN pip install .

# Debug server port
EXPOSE 5678

ADD run_smahub.sh /root/
RUN chmod +x /root/*
ENTRYPOINT ["/root/run_smahub.sh"]
```

run_smahub.sh
```
#!/bin/sh

cd /opt/smahub
python3 ./src/smahub.py -v
```

The run_smahub.sh uses an absolute path for python3 which seems to loose the venv.
To test and get it running I changed the git clone to a COPY.

Thank you!

**Comment by AnotherDaniel** [comment]
Hi @gandalfb, thank you for running smahub and reporting this. Pyhon is really not a very solid foundation it feels like... I'm looking into this!

**Comment by AnotherDaniel** [comment]
Alright, this should now be properly resolved, in release v1.5.2

**Comment by gandalfb** [comment]
Thank you!
Maybe docker python:alpine might be an alternative and provides some easy handling of venv. 

I will try the docker image from the repo later. Although it is working so nicely for now :-)
